src/fmt.rs

```diff
@@ -0,0 +1,28 @@
+use {
+    crate::{StrIndex, StrRange},
+    core::fmt,
+};
+
+impl fmt::Debug for StrIndex {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        fmt::Display::fmt(self, f)
+    }
+}
+
+impl fmt::Display for StrIndex {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.raw)
```

Comment:
Alignment is still lost here

Reply:
What do you mean? [This is works fine as far as I can tell.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0ed5ae7f08fc42ef2aefef337a4aa164)

Reply:
See https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f60485ee5d2f7c58440388bc764cc174

Reply:
Ah ok.